src/coremods/core_oper/core_oper.cpp

```diff
@@ -20,18 +20,6 @@
 #include "inspircd.h"
 #include "core_oper.h"
 
-namespace DieRestart
-{
-	bool CheckPass(User* user, const std::string& inputpass, const char* confentry)
```

Comment:
To avoid duplicating code this should be retained and `correctpass` and `hash` should be passed in as arguments.

Reply:
Both uses of this were replaced with `CommandDie::CheckPass()`, as `CommandRestart` subclasses `CommandDie`. Is there a usage I'm missing? I don't know of any duplicate code in this case

Reply:
Ah, I was misreading but looking at the code it seems like CheckPass is now literally just a call to PassCompare so it doesn't seem to make sense to do anything other than just inlining that one function call.

Reply:
Using the `inline` keyword would probably be beneficial, but I think leaving it as a separate method is better as if we wish to change the password logic for /DIE and /RESTART in the future we only have to modify `CommandDie::CheckPass()` instead of `CommanDie::Handle()` and `CommandRestart::Handle()`

Reply:
In the future there won't be *any* die or restart passwords. The current `<power>` config is just legacy compatibility for 2.0 configs and in 3.0+1 it will be removed entirely with just a check for the server name as done in Plexus and some other server implementations.

Reply:
Replaced